File: pythonProject/Servidor.py
class Servidor:
    def __init__(self, id=None, nombre=None, direccion_ip=None):
        self.id = id
        self.nombre = nombre
        self.direccion_ip = direccion_ip

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_nombre(self):
        return self.nombre

    def set_nombre(self, nombre):
        self.nombre = nombre

    def get_direccion_ip(self):
        return self.direccion_ip

    def set_direccion_ip(self, direccion_ip):
        self.direccion_ip = direccion_ip

    def __str__(self):
        return (f"Servidor{{id={self.id}, nombre='{self.nombre}', "
                f"direccionIp='{self.direccion_ip}'}}")

File: javaProject/co.edu.udec.s2.ejercicio30/src/domain/model/Servidor.java
package domain.model;

public class Servidor {
    private int id;
    private String nombre;
    private String direccionIp;
    
    public Servidor() {}
    
    public Servidor(int id, String nombre, String direccionIp) {
        this.id = id;
        this.nombre = nombre;
        this.direccionIp = direccionIp;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDireccionIp() {
        return direccionIp;
    }

    public void setDireccionIp(String direccionIp) {
        this.direccionIp = direccionIp;
    }

    @Override
    public String toString() {
        return "Servidor {" +
                "id=" + id +
                ", nombre='" + nombre + '\'' +
                ", direccionIp='" + direccionIp + '\'' +
                '}';
    }
}
